src/helpers/getStartUsers.ts

```diff
@@ -0,0 +1,28 @@
+import { users } from '@/constants/db';
+
+export const getDataFromLS = (key: string) => {
+  switch (key) {
+    case 'rememberedUsers': {
+      // @ts-ignore
```

Comment:
remove ts-ignore and come up with proper solution (maybe cast a type? e.g. const variable = JSON.parse(string) as Object)